src/covid_model_seiir_pipeline/pipeline/forecasting/model/containers.py

```diff
@@ -132,6 +132,32 @@ def to_dict(self) -> Dict[str, pd.DataFrame]:
         return utilities.asdict(self)
 
 
+@dataclass
+class PostprocessingParameters:
+    past_beta: pd.Series
+    past_compartments: pd.DataFrame
+
+    past_infections: pd.Series
+    past_deaths: pd.Series
+
+    infection_to_death: int
+    infection_to_admission: int
+    infection_to_case: int
+
+    ifr: pd.Series
+    ifr_hr: pd.Series
+    ifr_lr: pd.Series
+    ihr: pd.Series
+    idr: pd.Series
+
+    hospital_census: pd.Series
+    icu_census: pd.Series
+    ventilator_census: pd.Series
+
+    def to_dict(self) -> Dict[str, Union[int, pd.Series, pd.DataFrame]]:
+        return utilities.asdict(self)
```

Comment:
Not really related to this PR but ... do you ever plan on keeping track of which things are series vs data frames?

You're working with a tremendous number of "bags" of things. Really a `DataFrame` is a bag in of itself. Now you've got bags of bags of 

![image](https://user-images.githubusercontent.com/26555712/108136337-bd535280-706e-11eb-931c-9d0bbd9f4b29.png)

Reply:
I'm trying to get everything into series form.  It's going okay.  🤷